### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: testing/core-junit/src/main/java/io/ap4k/testing/annotation/Named.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package io.ap4k.testing.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * A name qualidier annotation.
```
Comment (geoand) [inline testing/core-junit/src/main/java/io/ap4k/testing/annotation/Named.java]: Typo

File: testing/core-junit/src/main/java/io/ap4k/testing/annotation/Named.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+@Target({ FIELD })
+@Retention(RUNTIME)
+public @interface Named {
+
+  /**
+   * The name of the annoated resouece.
```
Comment (geoand) [inline testing/core-junit/src/main/java/io/ap4k/testing/annotation/Named.java]: Typo

File: testing/openshift-junit/src/main/java/io/ap4k/testing/openshift/OpenshiftExtesnsion.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+@Internal
+public class OpenshiftExtesnsion implements ExecutionCondition, BeforeAllCallback, AfterAllCallback, WithPod, WithKubernetesClient, WithOpenshiftResources, WithProject, WithSourceToImageConfig {
```
Comment (geoand) [inline testing/openshift-junit/src/main/java/io/ap4k/testing/openshift/OpenshiftExtesnsion.java]: Typo

File: testing/kubernetes-junit/src/main/java/io/ap4k/testing/kubernetes/KubernetesExtension.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+public class KubernetesExtension implements  ExecutionCondition, BeforeAllCallback, AfterAllCallback,
+  WithPod, WithKubernetesClient, WithKubernetesResources, WithProject, WithDockerBuildConifg {
+
+
+  private static final String MANIFEST_PATH = "META-INF/ap4k/kubernetes.yml";
+  private static final String KUBERNETES_CONFIG_PATH = "META-INF/ap4k/.config/kubernetes.yml";
```
Comment (geoand) [inline testing/kubernetes-junit/src/main/java/io/ap4k/testing/kubernetes/KubernetesExtension.java]: This seems to be unused if I am not mistaked

File: testing/openshift-junit/src/main/java/io/ap4k/testing/openshift/OpenshiftExtesnsion.java (hunk 3 above)
Comment (iocanel) [reply testing/openshift-junit/src/main/java/io/ap4k/testing/openshift/OpenshiftExtesnsion.java]: Ohhhh its the extension....

Comment (geoand) [reply testing/openshift-junit/src/main/java/io/ap4k/testing/openshift/OpenshiftExtesnsion.java]: :P

**Pull request merged**